ui/apps/platform/src/Containers/Vulnerabilities/NodeCves/Overview/CVEsTable.tsx

```diff
@@ -1,48 +1,230 @@
+        nodeCVEs(query: $query, pagination: $pagination) {
+            cve
+            nodeCountBySeverity {
+                critical {
+                    total
+                }
+                important {
+                    total
+                }
+                moderate {
+                    total
+                }
+                low {
+                    total
+                }
+            }
+            topCVSS
+            affectedNodeCount
+            firstDiscoveredInSystem
+            distroTuples {
+                summary
+                operatingSystem
+                cvss
+                scoreVersion
+            }
+        }
+    }
+`;
+
+const totalNodeCountQuery = gql`
+    query getTotalNodeCount {
+        nodeCount
+    }
+`;
+
+// TODO Need to verify these types with the BE implementation
+export type NodeCVE = {
+    cve: string;
+    nodeCountBySeverity: {
+        critical: { total: number };
+        important: { total: number };
+        moderate: { total: number };
+        low: { total: number };
+    };
+    topCVSS: number;
+    affectedNodeCount: number;
+    firstDiscoveredInSystem: string;
+    distroTuples: {
+        summary: string;
+        operatingSystem: string;
+        cvss: number;
+        scoreVersion: string;
+    }[];
+};
+
 export type CVEsTableProps = {
     querySearchFilter: QuerySearchFilter;
     isFiltered: boolean;
     pagination: ReturnType<typeof useURLPagination>;
 };
 
-function CVEsTable({
-    /* eslint-disable @typescript-eslint/no-unused-vars */
-    querySearchFilter,
-    isFiltered,
-    pagination,
-    /* eslint-enable @typescript-eslint/no-unused-vars */
-}: CVEsTableProps) {
-    // TODO - Placeholders for query results
-    const data = [];
-    const previousData = undefined;
-    const error: Error | undefined = undefined;
-    const loading = false;
+function CVEsTable({ querySearchFilter, isFiltered, pagination }: CVEsTableProps) {
+    const { page, perPage } = pagination;
+    const { data, previousData, loading, error } = useQuery<
+        { nodeCVEs: NodeCVE[] },
+        {
+            query: string;
+            pagination: {
+                offset: number;
+                limit: number;
+            };
+        }
+    >(cvesListQuery, {
+        variables: {
+            query: getRegexScopedQueryString(querySearchFilter),
+            pagination: {
+                offset: (page - 1) * perPage,
+                limit: perPage,
+            },
+        },
+    });
+
+    const totalNodeCountRequest = useQuery<{ nodeCount: number }>(totalNodeCountQuery);
+    const totalNodeCount = totalNodeCountRequest.data?.nodeCount ?? 0;
 
     const tableData = data ?? previousData;
+    const tableState = getTableUIState({
+        isLoading: loading,
+        data: tableData?.nodeCVEs,
+        error,
+        searchFilter: querySearchFilter,
+    });
+
+    const expandedRowSet = useSet<string>();
+    const colSpan = 6;
```

Comment:
nit: Since it doesn't seem like we're ever modifying this number, this is a classic CONSTANT, and we should UPPER_CASE it's name.
```suggestion
    const COLUMN_SPAN = 6;
```